bundle.Dockerfile

```diff
@@ -5,7 +5,7 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=servicemeshoperator3
-LABEL operators.operatorframework.io.bundle.channels.v1="stable"
+LABEL operators.operatorframework.io.bundle.channels.v1="stable,stable-3.0"
```

Comment:
@FilipB In the main branch PR, we also have the following. 

operators.operatorframework.io.bundle.channel.default.v1: stable

Is it not required for the release branch?

Reply:
`operators.operatorframework.io.bundle.channel.default.v1: stable` is already there. It was added in earlier PR.

Reply:
Oh yeah. I missed that. Thanks.